src/test/kotlin/no/skatteetaten/aurora/boober/feature/AuroraAzureAppSubPartTest.kt

```diff
@@ -134,7 +136,22 @@ class AuroraAzureAppSubPartTest : AbstractMultiFeatureTest() {
             generateResources(
                 """{
              "azure" : {
-                "azureAppFqdn": "a.b.c.d" 
+                "azureAppFqdn": "valid.amutv.skead.no" 
+              }
+           }""",
+                createEmptyDeploymentConfig(), createdResources = 0
+            )
+        }
+    }
+
+    @ParameterizedTest
```

Comment:
Jeg ville byttet til en litt mer spesifikk test. Vi har noen metoder som hjelper litt. Jeg fjernet @foo fordi at den treffer to stk validation feil. FQDN validation og gyldig dns validation feil. 

```suggestion
        @ParameterizedTest
    @CsvSource("invalid", "invalid.notam.skead.no", "multiple.dots.amutv.skead.no", "invalid.skead.no")
    fun `it is an error if azureAppFqdn is not a valid fqdn for azure`(fqdn: String) {
        assertThat {
            createAuroraConfigFieldHandlers(
                """{
             "azure" : {
                "azureAppFqdn": "$fqdn",
                "groups": ["APP_dev", "APP_DRIFT"]
              }
           }"""
            )
        }.singleApplicationError("must be a fully qualified domain name ")
    }

```